src/python/bot/tasks/setup.py

```diff
@@ -604,19 +604,19 @@ def get_data_bundle_directory(fuzzer_name):
     logs.log_error('Unable to find fuzzer %s.' % fuzzer_name)
     return None
 
+  local_data_bundles_directory = environment.get_value('DATA_BUNDLES_DIR')
+
   # Check if we have a fuzzer-specific data bundle. Use it to calculate the
   # data directory we will fetch our testcases from.
   data_bundle = data_types.DataBundle.query(
       data_types.DataBundle.name == fuzzer.data_bundle_name).get()
   if not data_bundle:
     # Generic data bundle directory. Available to all fuzzers if they don't
     # have their own data bundle.
-    return environment.get_value('FUZZ_DATA')
+    return local_data_bundles_directory
```

Comment:
won't this break fuzzers that rely on web tests, since this is what's passed to the fuzzer --input_dir ? 

web tests are downloaded explicitly to FUZZ_DATA in update_task.py.

Reply:
ah right, forgot they will go into wrong dir. closing.